main.py

```diff
@@ -1,19 +1,30 @@
 from fastapi import Depends, FastAPI, HTTPException
-from datetime import date
-from typing import Dict
-from starlette.testclient import TestClient
+from fastapi.middleware.cors import CORSMiddleware
 
 from database.DatabaseResult import DatabaseResult
 from database.database import get_organization_uuid_from_hrpartner, get_interview_uuid_from_application_id, \
     get_user_from_interview_uuid
 from tests.test_db import override_get_interview_uuid_from_application_id, \
     override_get_organization_uuid_from_hrpartner, override_get_user_from_interview_uuid
+from additional_functions import is_valid_uuid
 
 app = FastAPI()
 
 
+origins = ["*"]
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=origins,
```

Comment:
Testowałeś to, bo to mi dość groźnie wygląda - jakieś authy, loginy, hasła XD

Reply:
Wziąłem z odpowiedzi na stacku XD

Reply:
Spoko, ale to żadnego hasła / tokenu / autoryzacji nie wymaga, nie?

Reply:
Nie chyba nie nic takiego nie pisali XD

Reply:
Okok